recipe/meta.yaml

```diff
@@ -12,12 +12,12 @@
 #       A Public Release for ISIS3.6.1:                        {% set version = "3.6.1" %}
 #       A Release Candidate for ISIS3.6.1:                     {% set version = "3.6.1_RC" %}
 #       A custom build of ISIS3.6.1 for the CaSSIS mission:    {% set version = "3.6.1_cassis" %}
-{% set version = "3.6.1" %}
+{% set version = "3.7.0_RC" %}
 
```

Comment:
I am just noting what you told me today - while semantic versioning requires hyphenation, we have to use an underscore in the meta.yml. :+1: